lib/client-assets.php

```diff
@@ -273,10 +273,12 @@ function gutenberg_register_vendor_script( $handle, $src, $deps = array() ) {
  * @param string $hook Screen name.
  */
 function gutenberg_scripts_and_styles( $hook ) {
-	if ( 'toplevel_page_gutenberg' !== $hook ) {
+	if ( ! preg_match( '/(toplevel|gutenberg)_page_gutenberg(-demo)?/', $hook, $page_match ) ) {
```

Comment:
Not 100% sure but somehow my browser tried loading all of the Gutenberg scripts when not on Gutenberg, when logging into WP-Admin. I haven't been able to reproduce yet, so I will check this out again in a bit.

Update: Can't reproduce, should be fine.